src/vite-env.d.ts

```diff
@@ -1 +1,2 @@
 /// <reference types="vite/client" />
+declare var config: any;
```

Comment:
Why is this "and" and not string?

Reply:
It should be an object/record since it might got something else in the future.

Reply:
We don't need to plan for the future. We can do that with tests. We need to define the object as it is today.

Reply:
Sure, thank you for this information.